test/topojson/test.ml

```diff
@@ -105,13 +105,60 @@ let get_foreign_members_in_point (f : Topojson.Topology.t) =
   | _ -> assert false
 
 let pp_ezjsonm ppf json = Fmt.pf ppf "%s" (Ezjsonm.value_to_string json)
+let tt = Topojson.Geometry.geometry
+(* let v = Topojson.Geometry.geometry_to_json in
+   Geometry.to_json v *)
+
+let pp ppf tt =
+  Fmt.pf ppf "%s"
+    (Ezjsonm.value_to_string (Topojson.Geometry.geometry_to_json tt))
```

Comment:
I think this just needs to be

```suggestion
    (Ezjsonm.value_to_string (Topojson.Geometry.to_json tt))
```

Reply:
I thought about doing that. But i get error ```
Unbound value Topojson.Geometry.to_json```

Reply:
Ah, https://github.com/geocaml/ocaml-topojson/pull/29 should fix this! Will merge once CI is happy.